ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/RecoveryRequired.java

```diff
@@ -0,0 +1,14 @@
+package com.arjuna.ats.jta.logging;
```

Comment:
I'm not sure about the placing the class under the `logging` package. As it informs about recovery processing it makes more sense to me to put it under `recovery` package. What was your reasoning about placing it here?

Reply:
Since this class is only used by jtaLogger for now, therefore I thought of placing it under `com.arjuna.ats.jta.logging` package would be fine for now. Maybe later if any other package requires it, then we could move it under ` recovery ` package?

Reply:
I will move it to `recovery` package